suite-native/storage/src/transforms/deviceTransforms.ts

```diff
@@ -0,0 +1,24 @@
+import { pipe, A } from '@mobily/ts-belt';
+import { createTransform } from 'redux-persist';
+
+import { TrezorDevice } from '@suite-common/suite-types';
+
+const serializeDevice = (device: TrezorDevice) => ({
+    ...device,
+    path: '',
+    remember: true,
+    connected: false,
+    buttonRequests: [],
+});
+
+export const devicePersistTransform = createTransform<TrezorDevice[], Readonly<TrezorDevice[]>>(
```

Comment:
nit: Graph slice handles its own persisting transformation but not device and wallet (I assume because it's wrapped inside wallet). Maybe we could also move graph here for consistency?

Reply:
I've tried that but ended with this solution because of the cyclic dependencies.

Reply:
What about moving graph transform here as well to keep them all together then?